src/main/AndroidManifest.xml

```diff
@@ -26,8 +26,5 @@
     android:versionCode="2"
     android:versionName="1.1" >
 
-    <uses-sdk
-        android:minSdkVersion="14"
-        android:targetSdkVersion="22" />
```

Comment:
build.grade appears to target version 23.  I'm not sure which one "wins" in that situation but it is possible that you have changed the behavior.